components/mpas-ocean/driver/ocn_comp_mct.F

```diff
@@ -2410,22 +2409,18 @@ subroutine ocn_import_mct(x2o_o, errorCode)!{{{
            end if
         endif
 
-        if ( landIceFreshwaterFluxField % isActive ) then
-           !landIceFreshwaterFlux(i) = x2o_o % rAttr(index_x2o_Fogx_qicelo, n)
-        end if
-        if ( landIceHeatFluxField % isActive ) then
-           !landIceHeatFlux(i) = x2o_o % rAttr(index_x2o_Fogx_qiceho, n)
+        if ( landIceFluxesCoupled ) then
+           landIceFreshwaterFlux(i) = x2o_o % rAttr(index_x2o_Fogx_qicelo, n)
+           landIceHeatFlux(i) = x2o_o % rAttr(index_x2o_Fogx_qiceho, n)
+           landIceInterfaceTracers(indexIT, i) = x2o_o % rAttr(index_x2o_Sg_blit, n)
+           landIceInterfaceTracers(indexIS, i) = x2o_o % rAttr(index_x2o_Sg_blis, n)
```

Comment:
Why does the ocean need this in the coupled case? Is this just present for debugging?

Reply:
Yes, we output them as part of the monthly means. This is for us but also has been requested by collaborators when we've shared our simulation data.